src/framework/application.js

```diff
@@ -1278,6 +1278,8 @@ pc.extend(pc, function () {
             // remove default particle texture
             pc.ParticleEmitter.DEFAULT_PARAM_TEXTURE = null;
 
+            pc.ModelHandler.resetDefaultMaterial();

```

Comment:
Would simply changing this line to pc.Scene.defaultMaterial.reset(); solve this issue? What do you think, @guycalledfrank?

Reply:
@willeastcott 
Can not solve this issue.

This issue is caused by old value(instance) remaining in global variable(pc.ModelHandler.DEFAULT_MATERIAL.parameters[_name_].scopeId).

This field is excluded from StandardMaterial.reset() method.